app/main.py

```diff
@@ -16,4 +16,15 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": [],
+}
```

Comment:
You should only populate lists for "mutable" and "immutable" keys here with variables, no other logic is required.